playbooks/artifact_upload.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Artifact uploader
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  vars:
+    region: "DFW"
+    cloud_name: "public_cloud"
+    component_metadata_file_path: "{{ lookup('env', 'WORKSPACE') }}/component_metadata.yml"
+    legacy_artifact_log_type:
+      - "type": "log"
+        "source": "{{ lookup('env', 'RE_HOOK_ARTIFACT_DIR') | default(lookup('env', 'WORKSPACE') ~ '/artifacts', true) }}"
+    artifact_type_include: "{{ (lookup('env', 'ghprbPullId') == '') | ternary('all', 'log') }}"
```

Comment:
Can you not use `artifact_type_include` here ?

Reply:
Well yes, I could just leave this out and not define it here - expecting it to be passed in. However, as @hughsaunders noted having it here is useful for testing outside of jenkins.

Reply:
OK, makes sense. Probably no point passing it in from groovy then?

Reply:
We pass it in from groovy for two reasons:

1. To allow our PR unit test to be able to upload more than just logs.
2. To allow any pipeline to set the type they want to upload as they wish, much like we want to for the unit test. :)